types/aws-lambda/trigger/lex-v2.d.ts

```diff
@@ -14,8 +14,13 @@ export interface LexV2Event {
     inputTranscript: string;
     bot: LexV2Bot;
     interpretations: LexV2Interpretation[];
-    requestAttributes: { [key: string]: string };
+    proposedNextState: {
+        dialogAction: LexV2DialogAction,
+        intent: LexV2Intent
+    };
+    requestAttributes?: Record<string, string>;
```

Comment:
To clarify: AWS will sometimes not send this input at all? This is a breaking change, so it should have good motivation (e.g. will fix a latent bug in user's code)

Reply:
That is correct, `requestAttributes` is sometimes absent entirely. The event that I pasted into `lex-vs-tests.ts` is a real example (with some info redacted) and doesn't have this attribute.

Reply:
Good change them!